packages/dashboard-frontend/webpack.config.common.js

```diff
@@ -19,12 +19,14 @@ const path = require('path');
 const config = {
   entry: {
     client: path.join(__dirname, 'src/index.tsx'),
+    'editor.worker': 'monaco-editor-core/esm/vs/editor/editor.worker.js'
   },
   output: {
     path: path.join(__dirname, 'lib'),
     publicPath: '/',
-    filename: 'client.[hash].js',
+    filename: '[name].bundle.js',
```

Comment:
Why did you drop `[hash]` out from the filename?

```suggestion
    filename: '[name].[hash].js',
```

Reply:
I need it because of editor worker:

```
MonacoEnvironment = {
  getWorkerUrl: () => './editor.worker.bundle.js'
};
```
https://github.com/eclipse-che/che-dashboard/pull/352/files#diff-7a4159f5a0b756c8b89cb914aceabe198e2fa7eeb60345e4a6d26a23e8080195R41

Reply:
I have prevented cashing with this code:
```
const doNotCache = [
...
    '/dashboard/client.bundle.js',
    '/dashboard/assets/branding/product.json',
  ];
...
```
https://github.com/eclipse-che/che-dashboard/pull/352/files#diff-5fcad887a01b080478e247668ff0bfe4c76f098de8bae61323ba21d4ebc4399cR37

Reply:
got it, thanks!